examples/README.md

```diff
@@ -13,3 +13,4 @@ Then, to run the examples, you can simply run the following commands:
 $ cd examples/
 $ python image_augmentation.py
 ```
+You can also view an interactive version of the examples on [Colab](./image_augmentation.ipynb)
```

Comment:
Nit: add a dot at the end of the sentence, i.e. `You can also view an interactive version of the examples on [Colab](./image_augmentation.ipynb).`